docs/reference/migration/migrate_6_1.asciidoc

```diff
@@ -1,8 +1,5 @@
 [[breaking-changes-6.1]]
 == Breaking changes in 6.1
-++++
```

Comment:
I think this is left over from a previous version?

Reply:
Good catch...